### DISCUSSION TITLE
GitHub Classroom Duplicate Assignment Bug

### DISCUSSION THREAD

**Discussion by ctnelson1997** [community/community#110900] [GitHub Education]
### Hi everyone,

Hi all,

I've run into a bug with creating an assignment on [GitHub classroom](https://classroom.github.com/), where, when pressing "Create Assignment" twice (usually because of speed/slowness) results in duplicate assignments.

<img width="475" alt="image" src="https://github.com/community/community/assets/37457808/10bfce31-20ca-4d3d-bfd2-4aff23a847ca">

<img width="584" alt="image" src="https://github.com/community/community/assets/37457808/bdce3c47-434b-4fe7-8846-bd31d5768f21">

These duplicate assignments cannot be accepted by students or deleted.

<img width="514" alt="image" src="https://github.com/community/community/assets/37457808/2e035ade-50e5-4cbb-be43-4c8d873316db">

My current workaround is just creating a new assignment (e.g. hw6-new), though this is an eyesore and creates URLs inconsistent with previous assignments. Any advice or fixes would be appreciated.

**Comment by JRDollar31** [comment]
We are having the same issue... And neither of the two could be removed.

![image](https://github.com/community/community/assets/106634844/e0793632-7fd7-420a-8f69-e760afac8744)

**Comment by lexolf** [comment]
Have the same problem after creating a Starter Assignment:
![image](https://github.com/community/community/assets/32224268/6dbb2f94-614e-48ea-a980-36a5633be0aa)

The links go to the same project page.

Attempts at deleting either of the assignments result in Error 406.

**Reply by lexolf** [reply]
Additional info: When deleting the assignment via "Edit" drop-down menu on its page, I get redirected to Classroom sign in page showing a "Cannot verify CSRF Token Authenticity" message.

**Comment by paninebaluy** [comment]
Same issue here. 
Cannot delete duplicate assignments, cannot edit them. The effects are the same as described above.

**Comment by CodyHenrichsen-CTEC** [comment]
Can neither create new assignments that students can access nor delete them. Auto duplicated.

**Reply by CodyHenrichsen-CTEC** [reply]
@samus-aran I have this same problem as well. I have tried removing the assignments on different machines and still no luck. Do you  know why it might be happening?

**Reply by ctnelson1997** [reply]
A bug on GitHub's end, still unaddressed.
